test/conformance/chainsaw/validating-policies/report/report.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: wgpolicyk8s.io/v1alpha2
+kind: PolicyReport
+metadata:
+  labels:
+    app.kubernetes.io/managed-by: kyverno
+  ownerReferences:
+  - apiVersion: apps/v1
+    kind: Deployment
+    name: nginx-deployment
+scope:
+  apiVersion: apps/v1
+  kind: Deployment
+  name: nginx-deployment
+results:
+- message: success
+  policy: check-deployment-labels
+  result: pass
+  rule: rule-0
+  scored: true
+  source: ValidatingPolicy
```

Comment:
I think the source should be KyvernoValidatingPolicy

Reply:
Why ?

Reply:
because the source field previously had Kyverno in it

ValidatingPolicy is pretty vague, KyvernoValidatingPolicy is unlikely to collide with other projects

Reply:
Dunno, we used "ValidatingAdmissionPolicy" for VAPs 🤷

Reply:
I don't have a strong opinion though

Reply:
Changed!